index.js

```diff
@@ -183,6 +185,54 @@ class GithubScm extends Scm {
         };
     }
 
+    /**
+     * Promise to wait a certain number of seconds
+     *
+     * Might make this centralized for other tests to leverage
+     *
+     * @method promiseToWait
+     * @param  {Number}      timeToWait  Number of seconds to wait before continuing the chain
+     * @return {Promise}
+     */
+    promiseToWait(timeToWait) {
+        return new Promise((resolve) => {
+            setTimeout(() => resolve(), timeToWait * 1000);
+        });
+    }
+
+    /**
+     * Wait computing mergeability
+     * @async  getPrMergeable
+     * @param  {Object}   config
+     * @param  {String}   config.scmUri     The scmUri to get PR info of
+     * @param  {String}   config.token      The token used to authenticate to the SCM
+     * @param  {Integer}  config.prNum      The PR number used to fetch the PR
+     * @param  {Integer}  count             The polling count
+     * @return {Promise}                    Resolves to object containing result of computing mergeability
+     *                                      The parameter of success exists for testing
+     */
+    async waitPrMergeability({ scmUri, token, prNum }, count) {
+        if (count >= POLLING_MAX_ATTEMPT) {
+            winston.warn('Computing mergerbility did not finish. '
```

Comment:
please change this to `logger.warn` see https://github.com/screwdriver-cd/scm-github/pull/146/files

Reply:
I fixed it.